### PR TITLE
Donate redesign

### PR DESCRIPTION
**Pull request by nanalital**

### CODE REVIEW

File: admin/app.rb [hunk 1]
```diff
@@ -83,15 +83,18 @@ class Admin < Padrino::Application
     end
 
     get :donate do
-      @title = "Donate"
+      @title = 'Donate'
       @supporter = Supporter.new
-      @web = true 
+      @web = true
+      @show_form ||= 1
```
Comment (nanalital) [inline admin/app.rb]: remove it . you can use @web. no?

File: admin/app.rb [hunk 2]
```diff
@@ -83,15 +83,18 @@ class Admin < Padrino::Application
...
+      @background_image = set_background_image
       render 'new', :layout => :web
     end
 
     get :new do
       @title = pat(:new_title, :model => 'supporter')
       @supporter = Supporter.new
+      @background_image = set_background_image
```
Comment (nanalital) [inline admin/app.rb]: remove + add @show_form = 1

File: admin/app.rb [hunk 3]
```diff
@@ -102,6 +105,7 @@ class Admin < Padrino::Application
       rescue => e
         p "Error in sending email to #{@sup}, exception => #{e}"
       end
+      @background_image = set_background_image
```
Comment (nanalital) [inline admin/app.rb]: remove

File: admin/app.rb [hunk 4]
```diff
@@ -201,8 +204,12 @@ class Admin < Padrino::Application
       @supporter = Supporter.new(params['supporter'])
       @supporter.acquired = Time.now
       amount = (@supporter.amount*100).to_s if @supporter.amount
-
+      p '====== inspecting ======'
+      p response.inspect
```
Comment (nanalital) [inline admin/app.rb]: remove

File: admin/app.rb [hunk 5]
```diff
@@ -201,8 +204,12 @@ class Admin < Padrino::Application
...
       if @supporter.save
+        p '=========================================='
+        p 'supporter.save successfully'
+        p '=========================================='
```
Comment (nanalital) [inline admin/app.rb]: remove

File: admin/app.rb [hunk 6]
```diff
@@ -215,7 +222,7 @@ class Admin < Padrino::Application
         request.add_field('Content-Type', 'application/x-www-form-urlencoded')
         request.body = "a="+amount+"&uniqnum="+@supporter.uniqnum+"&pfsAuthCode="+paymauth
         response = http.request(request)
-        #puts response.value
+        puts response.value
```
Comment (nanalital) [inline admin/app.rb]: back to comment

File: admin/app.rb [hunk 7]
```diff
@@ -231,7 +234,14 @@ class Admin < Padrino::Application
                     :pfsAuthCode=>paymauth,
                     :multi_settings_id=>"" }
           @verbose = nil#response.read_body
-          render 'redirect', :layout=>false
+
+          @background_image = set_background_image
+          # binding.pry
```
Comment (LitalFiverr) [inline admin/app.rb]: remove

**Pull request merged**